global/meta/models.yml

```diff
@@ -2806,6 +2807,26 @@ poll:
     read_only: true
     required: true
     restriction_mode: A
+  cryptographic_key:
+    type: string
+    description: base64 public key to cryptographic votes.
+    read_only: true
+    restriction_mode: A
+  cryptographic_key_signature:
```

Comment:
`crypt_signature`, analogue to votes_raw and votes_signature

Reply:
I don't see the analogy to `votes_raw` and `votes_signature`. What are your proposals for the names of these keys?

Reply:
votes_raw and votes_signature, but crypt_key and crypt_key_signature. In one case you strip the "usage" (raw), in the other case you leave the "usage" (key).